synapseclient/api/entity_services.py

```diff
@@ -964,3 +1245,305 @@ async def main():
 
     async for child in response:
         yield child
+
+
+async def set_entity_permissions(
```

Comment:
Do we have test for this function?

Reply:
This is being implicitly tested by the existing integration tests since it's used in the AccessControllable mixin